src/app/services/videoprocessingcuda/src/main.cpp

```diff
@@ -72,10 +74,12 @@ int main(int argc, char *argv[])
                                         dewarpingConfig,
                                         detectionDewarpingCount);
 
-        VideoThread videoThread(std::make_unique<VideoStream>(cameraConfig),
+        VideoThread videoThread(std::make_unique<CameraReader>(cameraConfig),
+                                //std::make_unique<ImageFileReader>("res/fisheye.jpg", ImageFormat::UYVY_FMT),

```

Comment:
Should we keep those 2 comments?

Reply:
I am not a fan of commenting code either, but this is the main for testing only so it's convenient to have them there.